### ISSUE TITLE
Organic with synthetic chest cant be blood filtered

### ISSUE THREAD

**Issue by projectkepler-ru** [tgstation/tgstation#82675]
## Issue Summary
You cannot blood filter an organic with synthetic chest, this is a problem if there is way too many toxin for them to be defbrillated

## Round ID:
N/A

## Testmerges:
N/A

## Reproduction:
This was tested locally that you can infact given someone over a certain amount of toxin  damage to kill them and they would not be revivable if they have synthetic chest due to the excessive amount of toxin not being removable via blood filter

**Comment by moocowswag** [comment]
Toxin damage doesnt prevent defib, just like oxygen.

**Comment by projectkepler-ru** [comment]
> Toxin damage doesnt prevent defib, just like oxygen.
> 
> But you are soft locked from removing chems from them unless you abuse pot water micro explosion to empty them.
> 
> I think the only (respected) way to safely remove chems from a dead body without blood filter is a virus with symptoms that lets it work on a dead person + metabolizes chems in them, then you can multiver as normal.


@moocowswag  No, this specifically means if you have toxin over 200, you cannot be defibbed at all and it can't be fixed because there isn't a way to heal toxin damage if your chest is synthetic
